frontend/src/v5/ui/routes/dashboard/projects/tickets/ticketsTable/ticketsTable.component.tsx

```diff
@@ -212,7 +213,7 @@ export const TicketsTable = () => {
 							TriggerButton={(
 								<NewTicketButton
 									startIcon={<AddCircleIcon />}
-									disabled={!containersAndFederations.length}
+									disabled={!containersAndFederations.length || getSelectedModelsNonViewerPermission().length === 0}
```

Comment:
I believe this causes a "Rendered more hooks than during the previous render" crash. I think swapping the conditions around might fix it. (I get this crash when I have a model selected and then I use the input to deselect all models)

The `getSelectedModelsNonViewerPermission().length === 0` part is broke though as it seems to always be true for me no matter what my permissions are. Also the button only appears disabled. It can still be clicked. I think you need another `disabled` prop on the parent NewTicketMenu component to disable the onClick